examples/camera/code.py

```diff
@@ -34,6 +37,41 @@
             last_frame, pycam.continuous_capture(), displayio.Colorspace.RGB565_SWAPPED
         )
         pycam.blit(last_frame)
+    elif pycam.mode_text == "LAPS":
+        if timelapse_remaining is None:
+            pycam._timelapsestatus_label.text = "STOP"
+        else:
+            timelapse_remaining = timelapse_timestamp - time.time()
+            pycam._timelapsestatus_label.text = f"{timelapse_remaining}s /    "
+        pycam._timelapse_rate_label.text = pycam._timelapse_rate_label.text
+        pycam._timelapse_submode_label.text = pycam._timelapse_submode_label.text
```

Comment:
Were these re-assignments of the label text really needed? I'll try testing without it.

Reply:
yes because otherwise display refresh doesnt pick up the labels after the screen has been blitted manually

Reply:
Makes sense, will add a comment to that effect